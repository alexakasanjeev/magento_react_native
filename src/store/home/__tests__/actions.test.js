import {
  getFeaturedProducts,
  getHomeConfigurableProductOptions,
} from '../actions';

describe('home actions', () => {
  test('getFeaturedProducts()', () => {
    // Setup
    const categoryId = '2';

    // Exercise
    const result = getFeaturedProducts(categoryId);

    // Verify
    expect(result).toMatchSnapshot();
  });

  test('getHomeConfigurableProductOptions()', () => {
    // Setup
    const sku = 'WH02-S';

    // Exercise
    const result = getHomeConfigurableProductOptions(sku);

    // Verify
    expect(result).toMatchSnapshot();
  });
});
