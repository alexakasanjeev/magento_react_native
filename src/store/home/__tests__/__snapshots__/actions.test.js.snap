// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home actions getFeaturedProducts() 1`] = `
Object {
  "payload": Object {
    "categoryId": "2",
  },
  "type": "MAGENTO_FEATURED_CATEGORY_PRODUCTS_REQUEST",
}
`;

exports[`home actions getHomeConfigurableProductOptions() 1`] = `
Object {
  "payload": Object {
    "sku": "WH02-S",
  },
  "type": "MAGENTO_HOME_UPDATE_CONF_PRODUCT_REQUEST",
}
`;
